### PR TITLE
telemetry helm chart

### PR DESCRIPTION
**Pull request by shorim**
**Description**

Changes proposed in this pull request:

- add `telemetry` helm chart with 2 sub-charts: `operator` and `fluent-bit`

**Related issue(s)**

#11822

### CODE REVIEW

**Review by JeremyHarisch — DISMISSED** [review]

File: resources/telemetry/charts/fluent-bit/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Fluent Bit Helm Chart
+
+[Fluent Bit](https://fluentbit.io) is a fast and lightweight log processor and forwarder or Linux, OSX and BSD family operating systems.
```
Comment (NHingerl) [inline resources/telemetry/charts/fluent-bit/README.md:3]:
```suggestion
[Fluent Bit](https://fluentbit.io) is a fast and lightweight log processor and forwarder for Linux, OSX, and BSD family operating systems.
```

File: resources/telemetry/charts/fluent-bit/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## Installation
+
```
Comment (NHingerl) [inline resources/telemetry/charts/fluent-bit/README.md:6]: are these instructions supposed to be done in order, or are they alternatives to each other?

**Review by NHingerl — CHANGES_REQUESTED** [review]

**Review by NHingerl — DISMISSED** [review]

**Review by JeremyHarisch — APPROVED** [review]

**Review by NHingerl — APPROVED** [review]

**Review by majakurcius — DISMISSED** [review]
No changes suggested, as it's third-party READMEs.

**Review by skhalash — DISMISSED** [review]

**Review by skhalash — APPROVED** [review]

**Review by majakurcius — APPROVED** [review]

**Review by JeremyHarisch — APPROVED** [review]

**Pull request merged**